test/integration_testcase.go

```diff
@@ -78,6 +78,9 @@ func (tc *IntegrationTestCase) Cleanup() {
 		j = bytes.Replace(j, cmds, n, -1)
 		j = append(j, '\n')
 		err = ioutil.WriteFile(filepath.Join(tc.rootPath, "testcase.json"), j, 0666)
+		if err != nil {
+			tc.t.Logf("Failed to update testcase %s: %s", tc.name, err)
+		}
```

Comment:
Yes - let's upgrade that to an `Errorf`. Especially because, if we don't, there's no guarantee the user would even see the log message (if the test otherwise passed).